packages/pulp/pulp.spec

```diff
@@ -507,6 +509,7 @@ if [ $1 -gt 1 ] ; then
         /sbin/service pulp_celerybeat stop > /dev/null 2>&1 || :
         /sbin/service pulp_resource_manager stop > /dev/null 2>&1 || :
     %endif
+    /usr/bin/chown -R apache:pulp /var/lib/pulp
```

Comment:
This is going to run on every upgrade which might become very slow if there's a lot files. Perhaps you should check if the user/group of `/var/lib/pulp` is already correct and otherwise recurse?

Reply:
@ekohl, thank you for the recommendation. I'll make a follow up PR.